transformer_engine/common/comm_gemm_overlap/comm_gemm_overlap.cpp

```diff
@@ -69,6 +77,30 @@ CommOverlapCore::CommOverlapCore(int myrank, int numranks, int mylocal, int numl
              num_comm_sm, set_sm_margin, use_ce, atomic_gemm);
 }
 
+CommOverlapCore::CommOverlapCore(int64_t nccl_comm_ptr, int tp_rank, int tp_size, int num_comm_sm,
+                                 bool is_p2p, bool atomic_gemm) {
+  _with_cublasmp = true;
```

Comment:
This should probably check if TE was built with cublasmp and error out otherwise.